lib/gems/pending/appliance_console.rb

```diff
@@ -307,7 +307,7 @@ module ApplianceConsole
             system_hosts.hostname = new_host
             system_hosts.set_loopback_hostname(new_host)
             system_hosts.save
-            LinuxAdmin::Service.new("network").restart
+            LinuxAdmin::Service.new("NetworkManager").restart
```

Comment:
@bdunne Thanks. `hostnamectl set-hostname name` works. And it's much faster than `systemctl restart network`. But the bug disappear today, `systemctl restart network` works and set hostname in ap won't crash. Do we still want to change that to `hostnamectl`? @carbonin